scripts/bash/postbuild.sh

```diff
@@ -13,17 +13,19 @@ echo "hello from postbuild.sh"
 #
 ####################################################################################
 
-apt-get install -y python3 python3-pip
+
 
 ####################################################################################
 #
 # Install Python3 modules used to scrape REDCap installation SQL script
 #
 ####################################################################################
-
+user=$(whoami)
+echo "Running with $user"
+curl -sS https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
Unclear why this is here? Seems like `pip` should already be installed?

Reply:
pip module is installed in root context. PIP module is not available in the build user's context.